app/index.js

```diff
@@ -111,6 +124,9 @@ module.exports = yeoman.generators.Base.extend({
         bower.dependencies.modernizr = '~2.8.1';
       }
 
+      if(Object.keys(bower.dependencies).length < 1) {
+        this.mkdir('bower_components');
+      }
```

Comment:
Why do we need this whole snippet? Won't bower automatically create the directory when the first package gets installed?